encryption_works.md

```diff
@@ -337,6 +337,18 @@ Washington Post journalist Barton Gellman, who Edward Snowden trusted with infor
 
 Snowden was right to be cautious and to insist that he check Gellman's new PGP fingerprint. PGP, if used right, provides the tools necessary to prevent MITM attacks. But these tools only work if the users are vigilant about identity verification.
 
+### What PGP Doesn't Do
+
+To use PGP effectively, it's good to understand what it does and doesn't do.
+
+For instance, using PGP with e-mail encryption only encrypts the *content* of your e-mails. It doesn't encrypt the to/from fields, the subject line or the metadata, the sundry details associated with the message. For an e-mail, metadata would include the associated e-mail addresses, the time and date sent and the IP address the e-mail originated from. This means that someone surveilling me can tell that at 2:05pm on Sunday, June 21, *tommy@freedom.press* e-mailed *trevor@freedom.press* from a certain IP address, but they can't tell what the content of the message was.
+
+Senator Diane Feinstein (D—Calif.) is a staunch defender of the NSA and has long [maintained](http://www.theguardian.com/world/2013/oct/21/dianne-feinstein-defends-nsa-data-collection) that the type of metadata collection the NSA carries out isn't intrusive because it doesn't include the content. Simply put, this isn't true, especially given that a caller's location is [included](http://www.theguardian.com/world/2013/jun/06/nsa-phone-records-verizon-court-order) in phone metadata collection. If you know that someone called a suicide hotline from the Golden Gate Bridge, you can infer something about their mental state. Similarly, the knowledge that a young woman called an abortion clinic is telling on its own -- you don't necessarily need to know what the content of the call was.
```

Comment:
I see what you're trying to do here, and agree that this is a good place to reference the ongoing debate over metadata vs. content collection. However, going from an example about PGP email to an example about phone conversations is a confusing. I don't know if readers should be expected to recognize why those two things are similar, or why debates about phone metadata may also apply to Internet metadata (especially since that is not a guarantee, now or in the future).